scripts/deploy_assisted_service.sh

```diff
@@ -73,6 +73,110 @@ elif [ "${DEPLOY_TARGET}" == "ocp" ]; then
 
     wait_for_url_and_run "$SERVICE_BASE_URL" "spawn_port_forwarding_command $SERVICE_NAME $OCP_SERVICE_PORT $NAMESPACE $NAMESPACE_INDEX $OCP_KUBECONFIG ocp $CLUSTER_VIP $SERVICE_NODEPORT"
     print_log "${SERVICE_NAME} can be reached at ${SERVICE_BASE_URL}"
+elif [ "${ENABLE_KUBE_API}" == "true" ]; then
+    echo "Installing Hive..."
+    cat << EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hive-operator
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: hive-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+    wait_for_operator "hive-operator" "openshift-operators"
+    wait_for_crd "clusterdeployments.hive.openshift.io"
+
+    add_firewalld_port ${OCP_SERVICE_PORT}
+    SERVICE_BASE_URL=https://${SERVICE_URL}:${OCP_SERVICE_PORT}
+
+    echo "Installing Assisted Installer Operator..."
+    cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+  labels:
+    name: ${NAMESPACE}
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: assisted-installer-group
+    namespace: ${NAMESPACE}
+spec:
+  targetNamespaces:
+    - ${NAMESPACE}
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: assisted-service-catalog
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: ${INDEX_IMAGE}
+  displayName: Assisted Test Registry
+  publisher: Assisted Developer
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: assisted-service-operator
+  namespace: ${NAMESPACE}
+spec:
+  installPlanApproval: Automatic
+  name: assisted-service-operator
+  source: assisted-service-catalog
+  sourceNamespace: openshift-marketplace
+  config:
+    env:
+    - name: SERVICE_IMAGE
+      value: ${SERVICE}
+    - name: OPENSHIFT_VERSIONS
+      value: ${OPENSHIFT_VERSIONS}
+    - name: SERVICE_BASE_URL
+      value: ${SERVICE_BASE_URL}
```

Comment:
use code from AS
I guess you'll need this as well:
https://github.com/openshift/assisted-service/pull/1618/files